### PR TITLE
Lint check

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -32,3 +32,9 @@ jobs:
```
Comment (al1img) [inline .github/workflows/ci.yaml:32]: Commit message Lint -> lint

File: certhandler/certhandler.go [hunk 2]
```diff
@@ -385,7 +385,7 @@ func (handler *Handler) Close() {
  ******************************************************************************/
```
Comment (al1img) [inline certhandler/certhandler.go:385]: Commit message: Lint -> lint

File: certhandler/modules/modules_test.go [hunk 3]
```diff
@@ -641,7 +636,11 @@ func TestTPMDictionaryAttackLockoutCounter(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Creating primary key failed: %v", err)
 	}
-	defer tpm2.FlushContext(tpmSimulator, handle)
+	defer func() {
+		if flushErr := tpm2.FlushContext(tpmSimulator, handle); flushErr != nil {
+			t.Errorf("Can;t flush contexst %s", flushErr)
```
Comment (al1img) [inline certhandler/modules/modules_test.go:641]: `Can't flush context: %s`

File: iamserver/iamserver.go [hunk 4]
```diff
@@ -437,7 +437,11 @@ func (server *Server) createServerProtected(cfg *config.Config, insecure bool) (
 	pb.RegisterIAMProtectedServiceServer(server.grpcServer, server)
 	pb.RegisterIAMPublicServiceServer(server.grpcServer, server)
 
-	go server.grpcServer.Serve(server.listener)
+	go func() {
+		if err := server.grpcServer.Serve(server.listener); err != nil {
+			log.Errorf("Can't serve grpc server: %s", err)
```
Comment (al1img) [inline iamserver/iamserver.go:442]: `Can't serve protected grpc server`

File: identhandler/modules/visidentifier/visidentifier.go [hunk 5]
```diff
@@ -299,7 +298,9 @@ func (instance *Instance) messageHandler(message []byte) {
 
 	switch header.Action {
 	case visprotocol.ActionSubscription:
-		instance.processSubscriptions(message)
+		if err := instance.processSubscriptions(message); err != nil {
+			log.Errorf("Failed process subscription %s", err)
```
Comment (al1img) [inline identhandler/modules/visidentifier/visidentifier.go:302]: `Failed to process subscription: %s`

File: identhandler/modules/visidentifier/visidentifier.go [hunk 6]
```diff
@@ -273,8 +272,8 @@ func (instance *Instance) handleConnection(url string) {
 
 		instance.wg.Done()
 
-		select {
-		case err := <-instance.wsClient.ErrorChannel:
+		{
```
Comment (al1img) [inline identhandler/modules/visidentifier/visidentifier.go:275]: Remove bracket

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Eabramov84) [reply .github/workflows/ci.yaml:32]: done

File: certhandler/modules/modules_test.go (hunk 3 above)
Comment (Eabramov84) [reply certhandler/modules/modules_test.go:641]: fixed

File: identhandler/modules/visidentifier/visidentifier.go (hunk 5 above)
Comment (Eabramov84) [reply identhandler/modules/visidentifier/visidentifier.go:302]: fixed

File: identhandler/modules/visidentifier/visidentifier.go (hunk 6 above)
Comment (Eabramov84) [reply identhandler/modules/visidentifier/visidentifier.go:275]: can't remove in this case we will have compilation error
identhandler/modules/visidentifier/visidentifier.go:260:9: goto reconnect jumps over declaration of err at identhandler/modules/visidentifier/visidentifier.go:275:7

File: certhandler/modules/swmodule/swmodule.go [hunk 7]
```diff
@@ -215,9 +215,8 @@ func (module *SWModule) ValidateCertificates() (
 			return nil, nil, nil, aoserrors.Wrap(err)
 		}
 
-		if _, ok := keyMap[key.Path]; ok {
-			delete(keyMap, key.Path)
-		}
+		delete(keyMap, key.Path)
+
```
Comment (al1img) [inline certhandler/modules/swmodule/swmodule.go:219]: Remove empty line.

Comment (Eabramov84) [reply certhandler/modules/swmodule/swmodule.go:219]: removed

**Review by al1img — APPROVED** [review]

**Pull request merged**